src/main/kotlin/game/racingcar/domain/RacingTrack.kt

```diff
@@ -0,0 +1,19 @@
+package game.racingcar.domain
+
+import game.racingcar.strategy.MoveStrategy
+
+class RacingTrack(private val cars: List<Car>) {
+    fun race(numberOfLap: Int, moveStrategy: MoveStrategy): List<Car> {
+        for (i in 0 until numberOfLap) {
+            cars.forEach { it.move(moveStrategy) }
+        }
+        return cars
+    }
+
+    fun getWinners(): List<Car> {
+        val maxPosition = cars.maxByOrNull { it.position }!!.position
+        return cars
+            .sortedByDescending { it.position }
+            .takeWhile { it.position == maxPosition }
```

Comment:
```suggestion
       return cars.filter { it.position == maxPosition }
```

`filter()`를 통하여 개선하는 방법도 있을 것 같아요.